src/background/permissions/permissions-ensurer.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { BrowserAdapter } from 'common/browser-adapters/browser-adapter';
+
+export class PermissionsEnsurer {
+    public static noPermissionsMessageEnding = 'Extension manifest must request permission to access this host.';
```

Comment:
Have you verified whether these messages get translated for users with non-english chrome language settings?